### PR TITLE
Support regex for both old and new tgen and oniontrace filenames

### PR DESCRIPTION
**Pull request by robgjansen**

### CODE REVIEW

File: tornettools/parse_oniontrace.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def parse_oniontrace_logs(args):
         logging.warning("Unable to parse oniontrace simulation data.")
         return
 
-    cmd_str = f"{otracetools_exe} parse -m {args.nprocesses} -e '.*\.oniontrace\.[0-9]+.stdout' shadow.data/hosts"
+    cmd_str = f"{otracetools_exe} parse -m {args.nprocesses} -e '.*\.oniontrace\.[0-9]+.stdout' -e 'stdout.*\.oniontrace\.[0-9]+.log' shadow.data/hosts"
```
Comment (sporksmith) [inline tornettools/parse_oniontrace.py:16]: A comment explaining why there are two patterns and which is which would be helpful.

File: tornettools/parse_tgen.py [hunk 2]
```diff
@@ -13,7 +13,7 @@ def parse_tgen_logs(args):
         logging.warning("Unable to parse tgen simulation data.")
         return
 
-    cmd_str = f"{tgentools_exe} parse -m {args.nprocesses} -e 'perfclient[0-9]+\.tgen\.[0-9]+.stdout' --complete shadow.data/hosts"
+    cmd_str = f"{tgentools_exe} parse -m {args.nprocesses} -e 'perfclient[0-9]+\.tgen\.[0-9]+.stdout' -e 'stdout.*perfclient[0-9]+\.tgen\.[0-9]+.log' --complete shadow.data/hosts"
```
Comment (sporksmith) [inline tornettools/parse_tgen.py:16]: Same here

**Review by sporksmith — APPROVED** [review]
Looks plausible. I'm assuming you tested locally and that these tools support multiple `-e` options :)

**Comment by robgjansen** [comment]
Yep, tested locally in both Shadow v1.15.0 and 2.0.0-pre.2, and it works with both filename formats. :)

File: tornettools/parse_oniontrace.py (hunk 1 above)
Comment (robgjansen) [reply tornettools/parse_oniontrace.py:19]: Done!

File: tornettools/parse_tgen.py (hunk 2 above)
Comment (robgjansen) [reply tornettools/parse_tgen.py:19]: Done!

**Pull request merged**